docs/content/mobile/android/cloud-deploy.md

```diff
@@ -7,30 +7,45 @@ Before reading this document, make sure you are familiar with:
 * [Development authentication flow](https://docs.improbable.io/reference/latest/shared/auth/development-authentication)
 * [Creating workers with WorkerConnector](https://docs.improbable.io/unity/alpha/content/gameobject/creating-workers-with-workerconnector)
 
+## Prepare your project to connect to a cloud deployment {#prepare}
+
+To connect your Android device to a cloud deployment, you need a mobile connector script.
+
+<%(Callout message="If you are using one of our [Starter Projects]({{urlRoot}}/content/glossary#starter-project), you can skip the **Create a mobile connector script** section below, as you already have one in your project.")%>
```

Comment:
maybe change to Note? think rob commented on it in the other PR that we try to avoid callouts

Reply:
fixed